admin_app/src/app/content/edit/page.tsx

```diff
@@ -30,17 +31,36 @@ const AddEditContentPage = () => {
   const content_id = Number(searchParams.get("content_id")) || null;
   const language_id = Number(searchParams.get("language_id")) || null;
   const [contentData, setContentData] = React.useState<{ [key: number]: Content }>({});
+  const [contentId, setContentId] = React.useState<number | null>(content_id);
+  const [languageId, setLanguageId] = React.useState<number | null>(language_id);
   const [content, setContent] = React.useState<Content | null>(null);
   const [isLoading, setIsLoading] = React.useState<boolean>(true);
-  const [refreshKey, setRefreshKey] = React.useState(0);
-  const [showDialog, setShowDialog] = React.useState(false);
-  const { token } = useAuth();
+  const [reloadTrigger, setReloadTrigger] = React.useState(0);
+  const { token, accessLevel } = useAuth();
+  const [snackMessage, setSnackMessage] = React.useState<string | null>(
+    null,
+  );
+  const getSnackMessage = (
+    action: string,
+    content_id: number | null,
+    language_id: number | null,
+  ): string | null => {
+    if (action === "delete") {
+      return `Content #${content_id} with language_id:#${language_id} deleted successfully`;
```

Comment:
So far, the content page doesn't communicate with language data. It would be easier to implement when we save languages to local storage. Would it be okay to add that as a separate ticket as well?

Reply:
Let's do that. Could you create a ticket for this and link to Sid's comment? Thanks!!